evalml/pipelines/utils.py

```diff
@@ -265,7 +270,9 @@ def _make_component_list_from_actions(actions):
         List of components used to address the input actions
```

Comment:
I think this needs to be updated from `list(ComponentBase)` to `list(tuple(ComponentBase, dict))`, correct?